doc/sphinx/source/input.rst

```diff
@@ -112,6 +112,13 @@ ESMValTool currently supports two ways to perform this reformatting (aka
    checks and fixes').  Details on this second method are given at the
    :ref:`end of this chapter <inputdata_native_datasets>`.
 
+A collection of readily CMORized OBS and OBS6 datasets can be accessed directly on CEDA/JASMIN and DKRZ. At CEDA/JASMIN
+OBS and OBS6 data is stored in the `esmeval` Group Workspace (GWS), and to be granted read (and execute) permissions to the
+GWS, one must apply at https://accounts.jasmin.ac.uk/services/group_workspaces/esmeval/ ; after permission has been granted, the user
+is encouraged to use the data in `/gws/nopw/j04/esmeval/obsdata-v2` locally, but not move it to minimize both data transfers and
```

Comment:
This repeats a path that is in the `config-user.yml` template. We should mention that or even link to it and consider even if this is needed here. Isn't at the end of the day the point of this bit of the documentation to say: "For the standard Jasmin config to work, you need to apply for this access?"

Reply:
removing it as we speaky, bud

Reply:
done in https://github.com/ESMValGroup/ESMValTool/pull/2385/commits/9fa931bcbf0fde645e2b02a1b41035acb460e9ba - rephrased a bit too, for literary convenience :grin: